airflow/config_templates/config.yml

```
@@ -116,6 +116,15 @@ core:
       type: string
       example: ~
       default: "16"
+    max_failure_runs_per_dag:
```

```suggestion
    max_consecutive_failed_dag_runs:
```

Would add "consecutive" for clarity